### ISSUE TITLE
Some breaklines in the examples of section 5.3.1.1 would greatly improve readability

### ISSUE THREAD

**Issue by jmaferreira** [DILCISBoard/E-ARK-CSIP#634]
<img width="1277" alt="Screenshot 2020-06-17 at 09 45 11" src="https://user-images.githubusercontent.com/3181302/84876741-7811f780-b07f-11ea-9146-5d8886001125.png">

<img width="1792" alt="Screenshot 2020-06-17 at 09 44 52" src="https://user-images.githubusercontent.com/3181302/84876750-7ba57e80-b07f-11ea-8bc4-1302fd287e7b.png">

**Comment by carlwilson** [comment]
Agreed that these are not ideal @jmaferreira . I've had a quick look at a (very) simple formatting algorithm that I think improves things but wrapping on attributes some and splitting `schemaLocation` across lines at space breaks. Again I'll get you eyes at a corrected PDF before 15/10/21
